radio/src/pulses/pulses_arm.cpp

```diff
@@ -144,18 +149,22 @@ void setupPulses(unsigned int port)
   switch (required_protocol) {
     case PROTO_PXX:
       setupPulsesPXX(port);
+      SCHEDULE_MIXER_START(port, 9);
```

Comment:
The heartbeat could be used to synchronize the pulses, that's one step further, here we synchronize the mixer with the pulses